docs/dg/settingUp.md

```diff
@@ -0,0 +1,57 @@
+<frontmatter>
+  title: "Setting up"
+  pageNav: 3
+</frontmatter>
+
+{% from 'scripts/macros.njk' import embed with context %}
+
+<include src="versionWarning.mbdf" />
+
+<h1 class="display-4">Setting up</h1>
+
+**Prerequisites:**
+* **JDK `1.8.0_60`** or later ([download :fas-download:](https://www.oracle.com/technetwork/java/javase/downloads/index.html)).
+* **npm** `6.0` or later ([download :fas-download:](https://www.npmjs.com/get-npm)).
```

Comment:
Is NPM an explicit pre-requisite? IIRC npm is installed as a plugin when running `gradle build`

Can the other devs help me to confirm this? Thanks

Reply:
> With this PR, the UG link in the pic below is now dead, since the `userguide.md` is removed. Should we just link to `reposense.org`?

reposense.org/ug will be the UG for the most recent release
reposense.org/RepoSense/ug will be the UG for the latest master